rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/20_terms.yml

```diff
@@ -724,7 +724,7 @@ setup:
         body: { "size" : 0, "aggs" : { "no_field_terms" : { "terms" : { "size": 1 } } } }
 
 ---
-"string profiler":
```

Comment:
I wrote this one so I figured I could fix it while I was here. Runtime fields don't have global ordinals so rather than skip the entire test for them I split it into two tests, one that runtime fields can run (below) and this one which runtime fields will skip.